build-test-workspace.xml

```diff
@@ -149,25 +161,21 @@
 				logging.file.path=${workspace.client.extension.dir}/logs
 			</echo>
 
-			<if>
-				<not>
-					<equals arg1="@{virtual.instance.id}" arg2="" />
-				</not>
-				<then>
-					<echo append="true" file="${workspace.client.extension.dir}/application.properties">
-						com.liferay.lxc.dxp.domains=@{virtual.instance.id}:8080
-						com.liferay.lxc.dxp.mainDomain=@{virtual.instance.id}:8080
-					</echo>
-				</then>
-			</if>
-
 			<print-file file.name="${workspace.client.extension.dir}/application.properties" />
 
 			<delete file="${workspace.client.extension.dir}/logs/spring.log" />
 
+			<echo><![CDATA[<gradle-execute dir="${workspace.client.extension.dir}" task="bootRun">
+	<arg value="-Pliferay.workspace.home.dir=${liferay.home}" />
+	<arg value="-PvirtualInstanceId=@{virtual.instance.id}" />
+</gradle-execute>]]></echo>
```

Comment:
is this just for debugging? Could we remove it?

Reply:
I don't even remember adding that!

Reply:
missed it again? :)

Reply:
Bah!